.github/workflows/pkgci_regression_test.yml

```diff
@@ -220,7 +220,7 @@ jobs:
             --goldentime-rocm-unet-ms 419.0 \
             --goldentime-rocm-clip-ms 18.5 \
             --goldentime-rocm-vae-ms 337.0 \
-            --goldendispatch-rocm-unet 1551 \
+            --goldendispatch-rocm-unet 1540 \
```

Comment:
Also, looks like these went down again (and some of the latencies too), so you can update these numbers.